diff --git a/dlls/xaudio2_7/xaudio_dll.c b/dlls/xaudio2_7/xaudio_dll.c
index 29de1d2cf5..6193128242 100644
--- a/dlls/xaudio2_7/xaudio_dll.c
+++ b/dlls/xaudio2_7/xaudio_dll.c
@@ -2527,7 +2527,7 @@ static BOOL xa2buffer_queue_period(XA2SourceImpl *src, XA2Buffer *buf, ALuint al
                 averr = avcodec_send_packet(src->conv_ctx, &avpkt);
                 if(averr){
                     WARN("avcodec_send_packet failed: %s\n", av_err2str(averr));
-                    break;
+//                    break;
                 }
 
                 buf->offs_bytes += avpkt.size;
